hw/ip/aes/aes.core

```diff
@@ -13,6 +13,7 @@ filesets:
       - lowrisc:ip:tlul
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:edn_pkg
+      - lowrisc:prim:edn_req
```

Comment:
Theoretically, this could go into `aes_env.core` because it should not be required to compile the RTL.

Reply:
@vogelpi this is not needed at all - and should be removed.
I added this when I asked you about the primitive and just forgot to remove it after you gave me the answer

Reply:
Ah, I see. Thanks for removing this @rasmus-madsen :+1: